[2024-12-05T23:10:16.151+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-05T23:10:16.185+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: job_listings_etl_v2.extract_csv manual__2024-12-05T23:10:13.078018+00:00 [queued]>
[2024-12-05T23:10:16.195+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: job_listings_etl_v2.extract_csv manual__2024-12-05T23:10:13.078018+00:00 [queued]>
[2024-12-05T23:10:16.197+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-12-05T23:10:16.214+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): extract_csv> on 2024-12-05 23:10:13.078018+00:00
[2024-12-05T23:10:16.231+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=752) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-05T23:10:16.237+0000] {standard_task_runner.py:63} INFO - Started process 757 to run task
[2024-12-05T23:10:16.240+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'job_listings_etl_v2', 'extract_csv', 'manual__2024-12-05T23:10:13.078018+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/ETL_FINAL_PROJECT (1).py', '--cfg-path', '/tmp/tmp9xz45k6g']
[2024-12-05T23:10:16.244+0000] {standard_task_runner.py:91} INFO - Job 28: Subtask extract_csv
[2024-12-05T23:10:16.345+0000] {task_command.py:426} INFO - Running <TaskInstance: job_listings_etl_v2.extract_csv manual__2024-12-05T23:10:13.078018+00:00 [running]> on host 1ed1f547130e
[2024-12-05T23:10:16.434+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='job_listings_etl_v2' AIRFLOW_CTX_TASK_ID='extract_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T23:10:13.078018+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T23:10:13.078018+00:00'
[2024-12-05T23:10:16.437+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-05T23:10:16.452+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-12-05T23:10:16.455+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.10.25-linuxkit-aarch64-with-glibc2.36
[2024-12-05T23:10:16.457+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-12-05T23:10:17.749+0000] {cursor.py:1149} INFO - Number of results in first chunk: 0
[2024-12-05T23:10:17.757+0000] {ETL_FINAL_PROJECT (1).py:143} INFO - Successfully extracted 0 records from HISTORICAL_DATA.
[2024-12-05T23:10:17.758+0000] {python.py:237} INFO - Done. Returned value was: []
[2024-12-05T23:10:17.759+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-05T23:10:17.786+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=job_listings_etl_v2, task_id=extract_csv, run_id=manual__2024-12-05T23:10:13.078018+00:00, execution_date=20241205T231013, start_date=20241205T231016, end_date=20241205T231017
[2024-12-05T23:10:17.871+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-05T23:10:17.914+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T23:10:17.917+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
